src/consent-manager/uploadConsentPreferences.ts

```diff
@@ -40,6 +40,10 @@ export const ManagedConsentDatabaseConsentPreference = t.intersection([
     prompted: t.union([t.literal('true'), t.literal('false')]),
     /** US Privacy (USP) String */
     usp: t.string,
+    /** IAB TCF String */
+    tcf: t.string,
+    /** IAB GPP String */
+    gpp: t.string,
```

Comment:
Are these fields returned for the read path as well?

Reply:
oh i missed the read path because it uses a completely different type for whatever reason... it seems to be pretty out of date too, let me fix that